File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for e in range(len(nums)):
            for i in range(e+1,len(nums)):
                if nums[e] + nums[i] == target:
                    return [e,i]
                
        

File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        Collection<Integer> seen = new ArrayList<>(nums.length);
        for (int i : nums){
            if (seen.contains(i)) {
                return true;}
            else{
                seen.add(i);}
        }
        return false;
        }
    }